MdeModulePkg/Library/DxeCorePerformanceLib/DxeCorePerformanceLib.c

```diff
@@ -21,7 +21,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 //
 // Data for FPDT performance records.
 //
-#define SMM_BOOT_RECORD_COMM_SIZE  (OFFSET_OF (EFI_SMM_COMMUNICATE_HEADER, Data) + sizeof(SMM_BOOT_RECORD_COMMUNICATE))
+#define SMM_BOOT_RECORD_COMM_SIZE  (OFFSET_OF (EFI_MM_COMMUNICATE_HEADER, Data) + sizeof(SMM_BOOT_RECORD_COMMUNICATE))

```

Comment:
looks the all changes in the DxeCorePerformanceLib.c belongs to the code refinement, no bug fix or function impact. if so, suggest use another commit for this.

Reply:
I moved the replacement of `EFI_SMM_COMMUNICATE_HEADER` with `EFI_MM_COMMUNICATE_HEADER` and the associated variable name to its own commit.